users/frobiac/layout.h

```diff
@@ -0,0 +1,274 @@
+// Copyright 2023 @frobiac
+// SPDX-License-Identifier: GPL-2.0-or-later
+//
+#pragma once
+
+// has to be copied from QMK, but without "#include keymap.h" to be able to build from keyboards
+#include "keymap_german.h"
+
+// Contains a layout based on 'Aus der Neo-Welt', 'NordTast' and 'Neo' extra layers
+
+// @TODO Based on german keymap
+
+// Support
+// 4x12  : HyperNano (2x2 unused)
+// 3x5_3 : CrowBoard, cantor
+// 3x5_2 : Sweep
+// 4x5   : BlackBowl (3 thumbs, 1 extra upper right,  2x1 unused ring finger)
+// 4x6   : RedTilt (see BlackBowl plus extra column)
+
+/*
+Since our quirky block definitions are basically a list of comma separated
+arguments, we need a wrapper in order for these definitions to be
+expanded before being used as arguments to the LAYOUT_xxx macro.
+*/
+#if (!defined(LAYOUT) && defined(KEYMAP))
+#    define LAYOUT KEYMAP
+#endif
+
+// clang-format off
+
+#define LAYOUT_hypernano_4x12_wrapper(...)  LAYOUT_hypernano_4x12(__VA_ARGS__)
+#define LAYOUT_blackflat_4x12_wrapper(...)  LAYOUT_blackflat_4x12(__VA_ARGS__)
+#define LAYOUT_sweep_3x5_2_wrapper(...)     LAYOUT_sweep_3x5_2(__VA_ARGS__)
+#define LAYOUT_crowboard_3x5_3_wrapper(...) LAYOUT_crowboard_3x5_3(__VA_ARGS__)
+
+
+// Copyright 2023 frobiac
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+// @TODO created variant on US OS layout. Shows up nicer in tools and web editor, if json is created from it
+// @TODO refactor into generic AdNW layer for use in other keyboards
+// @TODO clear RHS, modifiers in mouse layer
+
+#define _FROBIAC 0
+#define _QWERTZ  1
+#define _SYMBOLS 2
+#define _NAVNUM  3
+#define _FUNC    4
+#define _MOUSE   5
```

Comment:
```suggestion
enum layers {
    _FROBIAC,
    _QWERTZ,
    _SYMBOLS,
    _NAVNUM,
    _FUNC,
    _MOUSE
};
```

Reply:
Indeed, after adding default keymap configs per board as you suggested and setting the username in rules.mk, all issues with separate keymap_german.h and also this enum vs. xprintf compile are gone.
Thx!